build/build-image/cross/Makefile

```diff
@@ -21,7 +21,7 @@ TAG=$(shell cat VERSION)
 all: push
 
 build:
-	docker build --pull -t gcr.io/google_containers/$(IMAGE):$(TAG) .
+	docker build --pull -t k8s.gcr.io/$(IMAGE):$(TAG) .
 
 push: build
-	gcloud docker --server=gcr.io -- push gcr.io/google_containers/$(IMAGE):$(TAG)
+	gcloud docker --server=gcr.io -- push k8s.gcr.io/$(IMAGE):$(TAG)
```

Comment:
--server=k8s.gcr.io

Reply:
Can you comment on the need for this?  I don't understand this flag